exam001/Views/ViewController.swift

```diff
@@ -0,0 +1,482 @@
+//
+//  ViewController.swift
+//  exam001
+//
+//  Created by Adnan Al-Akhrass on 11/8/18.
+//  Copyright © 2018 Adnan Al-Akhrass. All rights reserved.
+//
+
+//
+
+import UIKit
+import Firebase
+
+enum textfieldIndex: Int {
+    case first
+    case second
+    case third
+    case fourth
+}
+
+struct Requestes {
+    var fullName: String
+    var floor: String
+    var location: String
+    var items: String
+}
+
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, FireBaseModelDelegate {
```

Comment:
`class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, FireBaseModelDelegate`

Each conformance should be separated on the same file as extensions
Example:
extension ViewController: UITableViewDelegate {
    // Include related methods here
}

// Other extensions goes here

Reply:
noted, however the tableview delegate and datasource are also called in separate extensions?